draft-ietf-jsonpath-base.md

```diff
@@ -1370,12 +1370,12 @@ Queries:
 
 ## Semantics of `null`
 
-Note that JSON `null` is treated the same as any other JSON value: it is not taken to mean "undefined" or "missing".
+Note that JSON `null` is treated the same as any other JSON: it is not taken to mean "undefined" or "missing".
```

Comment:
I think this should be "as any other JSON value" here.

Reply:
Whoops. Thanks.